src/ontology/uberon-edit.obo

```diff
@@ -224906,6 +224906,35 @@ intersection_of: part_of UBERON:0003889 ! fallopian tube
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-10-21T08:55:11Z xsd:dateTime
 
+[Term]
+id: UBERON:8480038
+name: meningeal myeloid tissue
+def: "Lymphomyeloid tissue that consist mainly in granulopoietic cells but that also contains erythroid and lymphoid cells in different stages of maturation." [PMID:25130144]
+is_a: UBERON:0034769 ! lymphomyeloid tissue
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+relationship: in_taxon NCBITaxon:7898 ! Actinopterygii
+property_value: curator_notes "May be present in Amia and Lepisosteus, Polyodon and Acipenser, but not Polypterus." xsd:string {xref="https://doi.org/10.1111/j.1096-3642.1998.tb02524.x"}
+property_value: dcterms-date 2022-10-31T07:50:56Z xsd:dateTime
+
+[Term]
+id: UBERON:8480039
+name: milt
+def: "Seminal fluid of fish, mollusks, and certain other water-dwelling animals which reproduce by spraying this fluid, which contains the sperm, onto the egg cells." [https://en.wikipedia.org/wiki/Milt]
+is_a: UBERON:0006536 ! male reproductive gland secretion
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+relationship: in_taxon NCBITaxon:6447 ! Mollusca
+relationship: in_taxon NCBITaxon:7898 ! Actinopterygii
+property_value: dcterms-date 2022-10-31T07:54:42Z xsd:dateTime
+
+[Term]
+id: UBERON:8480040
+name: ovarian fluid
+def: "Fish ovary secretion released with eggs." [https://hal.archives-ouvertes.fr/hal-00899767/document]
```

Comment:
note on definition source: We always prefer IDs like PMIDs or URLs of primary literature databases rather than secondary archives.

In this case, the relevant URL is
https://pubmed.ncbi.nlm.nih.gov/8526977/

or simply PMID:8526977

Reply:
is ovarian fluid only secreted in fish?

Reply:
@cmungall I do not know if ovarian fluid is only secreted in fish
@cmungall I will make the change about the reference